Examples/create-app.js

```diff
@@ -0,0 +1,128 @@
+let appNameIOS = `${appName}-ios`;
+let reactNativeVersion = args[1] || `react-native@${execSync('npm view react-native version')}`.trim();
+let reactNativeCodePushVersion = args[2] || `react-native-code-push@${execSync('npm view react-native-code-push version')}`.trim();
+
+console.log(`App name: ${appName}`);
+console.log(`React Native version: ${reactNativeVersion}`);
+console.log(`React Native Module for CodePush version: ${reactNativeCodePushVersion} \n`);
+
+try {
+    console.log(`Creating "${appNameAndroid}" to release updated for Android...`);
+    execSync(`code-push app add ${appNameAndroid} android react-native`);
+    console.log(`App "${appNameAndroid}" has been created \n`);
+} catch (e) {
+    console.log(`App "${appNameAndroid}" already exists \n`);
+}
+let androidDeploymentKeys = JSON.parse(execSync(`code-push deployment ls ${appNameAndroid} -k --format json`));
+let androidStagingDeploymentKey = androidDeploymentKeys[1].key;
+console.log(`Deployment key for Android: ${androidStagingDeploymentKey}`);
+console.log(`Use "code-push release-react ${appNameAndroid} android" command to release updates for Android \n`);
+
+try {
+    console.log(`Creating "${appNameIOS}" to release updated for iOS...`);
+    execSync(`code-push app add ${appNameIOS} ios react-native`);
+    console.log(`App "${appNameIOS}" has been created \n`);
+} catch (e) {
+    console.log(`App "${appNameIOS}" already exists \n`);
+}
+let iosDeploymentKeys = JSON.parse(execSync(`code-push deployment ls ${appNameIOS} -k --format json`));
+let iosStagingDeploymentKey = iosDeploymentKeys[1].key;
+console.log(`Deployment key for iOS: ${iosStagingDeploymentKey}`);
+console.log(`Use "code-push release-react ${appNameIOS} ios" command to release updates for iOS \n`);
+
+console.log(`Installing React Native...`);
+execSync(`react-native init ${appName} --version ${reactNativeVersion}`);
+console.log(`React Native has been installed \n`);
+
+process.chdir(appName);
+
+console.log(`Installing React Native Module for CodePush...`);
+execSync(`npm i --save ${reactNativeCodePushVersion}`);
+console.log(`React Native Module for CodePush has been installed \n`);
+
+console.log(`Linking React Native Module for CodePush...`);
+nexpect.spawn(`react-native link react-native-code-push`)
+    .wait("What is your CodePush deployment key for Android (hit <ENTER> to ignore)")
+    .sendline(androidStagingDeploymentKey)
+    .wait("What is your CodePush deployment key for iOS (hit <ENTER> to ignore)")
+    .sendline(iosStagingDeploymentKey)
+    .run(function (err) {
+        if (!err) {
+            console.log(`React Native Module for CodePush has been linked \n`);
+            setupAssets();
+        }
+        else {
+            console.log(err);
+        }
+    });
+
+function setupAssets() {
+    fs.unlinkSync('./index.ios.js');
+    fs.unlinkSync('./index.android.js');
+
+    fs.createReadStream('../CodePushDemoApp/demo.js').pipe(fs.createWriteStream('demo.js'));
```

Comment:
@sergey-akhalkov yeah, streams are working asynchronously and should be processed respectively. Usage of writeFileSync/readFileSync are the simplest alternative to copy files in this particular case.